merlin.psi.ch/cpu/codes/phonopy-2.20.0.yml

```diff
@@ -0,0 +1,19 @@
+---
+label: '{{ code_binary_name }}-2.20.0'
+computer: merlin6-cpu
+default_calc_job_plugin: phonopy.phonopy
+description: The phonopy code containerized mainly for merlin-psi cluster, CPU partition.
+engine_command: apptainer exec {image_name}
+filepath_executable: /opt/conda/bin/phonopy
+image_name: docker://mikibonacci/phonopy-2.20.0:amd64
+prepend_text: export APPTAINER_BINDPATH=$PWD
+append_text: ''
+use_double_quotes: false
+wrap_cmdline_params: false
+metadata:
+    template_variables:
+        code_binary_name:
+            key_display: Code name
+            type: list
+            options:
+                - phonopy
```

Comment:
```suggestion
```

Reply:
Since it is only one code, I didn't see why need use list, it will render as a dropdown in widget.